extensions/wikia/Email/i18n/crowdin-watchedpage.conf

```diff
@@ -0,0 +1,5 @@
+[crowdin.project]
+projectid = "ext-Email-WatchedPage"
+format = "mediawiki"
+source.file = "WatchedPage.i18n.php"
+repo.branch = SOC-558
```

Comment:
I don't think we should add this.  The only thing this does is allow crowdin to automatically check out the specified branch and commit the changes to it.  If we really used this we'd have to change and commit it every time we updated the file. Better to only download from crowdin, and do the commit and push ourselves.